### ISSUE TITLE
AuthorRetrieval is missing some papers

### ISSUE THREAD

**Issue by AndreaUnige** [pybliometrics-dev/pybliometrics#159]
When I perform the "AuthorRetrieval" some products are not returned.

For example, if I do `AuthorRetrieval(scopusID)` , putting my personal ScopusID I get the list of my papers.
Unfortunately, in this list, some papers are missing while If I check on my Scopus page I can find them.

This is not always true for some of my co-authors.
This means that if I do `AuthorRetrieval(scopusID)` putting one of my co-authors' ScopusId, then the missing papers appear in the returned list.

In other words, it seem that the code `AuthorRetrieval(scopusID)` sometimes does not return all the products associated to the specified _scopusID_. It depends on the ScopusID.

Am I missing something or this is a known issue ?

**Comment by Michael-E-Rose** [comment]
Hi @AndreaUnige ! Unless you give me your Scopus ID I cannot look into the issue.

A first guess however is that your data is not up-to-date. Try using the `refresh` parameter, e.g. setting it to `True`. When you have an `AuthorRetrieval()` object, you can call `.get_cache_file_age()` on it to see the age of the data in your local cache.

**Comment by AndreaUnige** [comment]
Dear @Michael-E-Rose,
many thanks for your quick reply. Please allow me to give you the details.

- My ScopusID is: 36180166900
- My co-author ScopusID is: 57053502600

Now, if you try the command `AuthorRetrieval(36180166900)`, so using my ScopusID you will get 52 documents. However, if you look at my Scopus page (see here: https://www.scopus.com/authid/detail.uri?origin=resultslist&authorId=36180166900&zone=) you can find 55 papers (I double-checked this again few seconds ago). Therefore, 3 papers missing.

After a manual comparison, I have found out the papers are:

1. A Wearable Prototype for Neurological Symptoms Recognition
2. A Tomographic Multistatic System for Biomedical Microwave Sensing
3. Computational Complexity Closed-Form Upper Bounds Derivation for Fingerprint-Based Point-of-Interest Recognition Algorithms

However, if I tried the same procedure with one of my co-authors with ScopusID: 57053502600 (who is co-author only of papers 1. and 3.) I got the ALL the papers present on her Scopus page which are 20 (see here: https://www.scopus.com/authid/detail.uri?authorId=57053502600)
This also includes papers 1. and 3. reported above.

Finally, I already set the `refresh` parameter to true but nothing changes.
Let me know if you need further information.

**Comment by Michael-E-Rose** [comment]
I cannot replicate the behavior you're describing:

```
>>> import pybliometrics
>>> pybliometrics.__version__
'2.6.3'
>>> from pybliometrics.scopus import AuthorRetrieval
>>> au = AuthorRetrieval(36180166900)
>>> print(au)
Sciarrone A. from DITEN in Italy,
published 55 document(s) since 2010 
which were cited by 500 author(s) in 654 document(s) as of 2020-09-17
>>> au.document_count
'55'
>>> docs = au.get_documents()
>>> len(docs)
55
>>> len(set([p.eid for p in docs]))
55
```

So, all the time I get 55 documents for you. Where _exactly_ do you find only 52?
